{{cookiecutter.project_name}}/docs/template_usage.md

```diff
@@ -419,6 +419,44 @@ The following hints may be useful to work with the template sync:
 You have reached the end of this document. Congratulations! You have successfully set up your project and are ready to start.
 For everything else related to documentation, code style, testing and publishing your project to pypi, please refer to the [contributing docs](contributing.md#contributing-guide).
 
+## Migrate existing projects to using this template
+
+You can also update existing projects to make use of this template to benefit from the latest-greatest
+tooling and automated template updates. This requires some manual work though. Here's one way how to do it
+
+1. Let's assume your repository is checked out to `$REPO`
+2. Clone your repository a second time to `${REPO}_cookiecutterized`
+3. Initialize an empty repository from this cookiecutter template:
+
+    ```bash
+    mkdir template && cd template
+    cruft create cruft create https://github.com/scverse/cookiecutter-scverse
+    ```
+
+4. remove everything from the existing repo
+
+    ```bash
+    cd ${REPO}_cookiecutterized
+    git co -b cookiecutterize
```

Comment:
```suggestion
    git switch -c cookiecutterize
```

Reply:
it's my personal alias. Thanks for spotting it!